src/EventStore.Core/Services/Archiver/Storage/IArchiveStorage.cs

```diff
@@ -2,17 +2,39 @@
 // Event Store Ltd licenses this file to you under the Event Store License v2 (see LICENSE.md).
 
 using System;
+using System.Collections.Generic;
+using System.Threading;
 using System.Threading.Tasks;
+using EventStore.Core.Services.Archiver.Storage.Exceptions;
 
 namespace EventStore.Core.Services.Archiver.Storage;
 
 public interface IArchiveStorage {
-	public static IArchiveStorage None = new NoArchiveStorage();
-	public ValueTask StoreChunk(string path);
-}
+	/// <summary>Stores a chunk in the archive</summary>
+	/// <param name="chunkPath">The path of the chunk to archive</param>
+	/// <param name="ct">The token to monitor for cancellation requests</param>
+	/// <exception cref="ChunkDeletedException">Thrown if the chunk file is deleted while being archived</exception>
+	/// <exception cref="OperationCanceledException">Thrown if cancellation was requested</exception>
```

Comment:
open to opinions on this, but personally i dont think its worth it to document the meaning of the cancellation token and the possibility of operationcanceledexception when they're the normal pattern

Reply:
yup, fixed :+1: